networks/mainnet/rpcs.md

```diff
@@ -4,48 +4,170 @@ description: Public RPC endpoints are available for the Filecoin mainnet.
+    </td>
+    <td>
+      <code>wss://wss.node.glif.io/apigw/lotus/rpc/v1</code>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      Docs
+    </td>
+    <td>
+      <a href="https://api.node.glif.io">Glif Nodes - Docs</a>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      API Support
+    </td>
+    <td>
+      All Filecoin and Eth JSON RPC methods including <a href="https://docs.filecoin.io/reference/json-rpc/mpool#mpoolpush">MPoolPush</a>
+    </td>
+  </tr>
+  <tr>
+   <td>
+      Contact
+    </td>
+    <td>
+        <a href='https://filecoinproject.slack.com/archives/C017HM9BJ8Z'>#fil-glif-node-hosting</a> in <a href='https://filecoin.io/slack'>Filecoin Slack</a>
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="4">
+      <a href="https://ankr.com">Ankr</a>
+    </td>
+    <td>
+      HTTPS
+    </td>
+    <td>
+      <code>https://rpc.ankr.com/filecoin</code>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      WSS
+    </td>
+    <td>
+      By request at: https://www.ankr.com/rpc/filecoin
+    </td>
+  </tr>
+  <tr>
+    <td>
+      Docs
+    </td>
+    <td>
+      <a href="https://www.ankr.com/docs/rpc-service/chains/chains-list/#filecoin">Ankr Docs - Filecoin</a>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      API Support
+    </td>
+    <td>
+      All Filecoin and Eth JSON RPC methods including <a href="https://docs.filecoin.io/reference/json-rpc/mpool#mpoolpush">MPoolPush</a>
+    </td>
+  </tr>  
+  <tr>
+    <td rowspan="3">
+      <a href="https://cloud.chainup.com/">Chainup Cloud</a>
+    </td>
+    <td>
+      HTTPS
+    </td>
+    <td>
+      <code>https://filecoin.chainup.net/rpc/v1</code>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      WSS
+    </td>
+    <td>
+      <code>wss://filecoin.chainup.net/rpc/v1</code>
+    </td>
+  </tr>
+  <tr>
+    <td>
+      Docs
+    </td>
+    <td>
+      <a href="https://docs.chainupcloud.com/blockchain-api/filecoin/public-apis">Chainup Cloud Docs - Filecoin</a>
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="2">
+      <a href="https://nownodes.io/">NOWNodes</a>
+    </td>
+    <td>
+      HTTPS
+    </td>
+    <td>
+      <code>https://fil.nownodes.io</code> (Free for 1 month with <a href="https://nownodes.io/pricing">signup</a>)
+    </td>
+  </tr>
+  <tr>
+    <td>
+      Docs
+    </td>
+    <td>
+      <a href="https://documenter.getpostman.com/view/13630829/TVmFkLwy">NOWNodes - Docs</a>
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="2">
+      <a href="https://getblock.io/nodes/fil">GetBlock</a>
+    </td>
+    <td>
+      HTTPS
+    </td>
+    <td>
+      <code>https://filecoin.getblock.io</code> (Free with <a href="https://getblock.io/nodes/fil/">signup</a>)
+    </td>
+  </tr>
+  <tr>
+    <td>
+      Docs
+    </td>
+    <td>
+      <a href="https://getblock.io/docs/getblock-explorer/get-started/">GetBlock - Docs</a>
+    </td>
+  </tr>
+</table>
+
+
+## Additional Notes:
+
+*  Glif Nodes offers a [Lotus Lite node](https://lotus.filecoin.io/lotus/install/lotus-lite/) endpoint with the following command:
```

Comment:
```suggestion
*  Glif Nodes offers a [Lotus Lite node](https://docs.filecoin.io/nodes/lite-nodes/spin-up-a-lite-node) endpoint with the following command:
```
The docs.filecoin.io version of the lite-node installation provides a more comprehensive and up-to-date guide.